api/client.go

```diff
@@ -229,6 +229,7 @@ func (c Client) ListIdentityGrants(id uid.ID) (*ListResponse[Grant], error) {
 	return list[ListResponse[Grant]](c, fmt.Sprintf("/v1/identities/%s/grants", id), nil)
 }
 
+// Deprecated: use ListGroups
```

Comment:
Should we create an issue to track removing this?

Reply:
It is added to https://github.com/infrahq/infra/issues/1868, I think we can use a single issue to track all of it.

Reply:
We already effectively removed it in the rename from identity to user, so I went ahead and removed it. No reason to deprecate a method that was never in a release.